wesnoth-did.po

```diff
@@ -1454,25 +1464,33 @@ msgid ""
 "is stronger than you, it will break free of your hold. To master it, your "
 "own soul must be unwavering."
 msgstr ""
+"Ovládnutí duchů, zvláště těch mocnějších, je něco, s čím se mohou potýkat i "
+"zkušení nekromanti. Zatímco ostatní nemrtví nejsou po oživení ničím jiným "
+"než bezduchými otroky, duchové jsou dušemi svých původních majitelů, a tak "
+"si zachovávají určitý stupeň vědomí. Pokud je duch silnější než ty, vymaní "
+"se z tvé kontroly. Abys ho ovládl, musí být tvoje vlastní duše neochvějná."
 
 #. [message]: speaker=Malin Keshar
 #: data/campaigns/Descent_Into_Darkness/scenarios/03_A_Haunting_in_Winter.cfg:1529
 msgid "I doubt I shall be overcome by the soul of an orc."
-msgstr ""
+msgstr "Pochybuji, že by se mnou mohla soupeřit nějaká skřetí duše."
 
 #. [message]: speaker=Darken Volk
 #: data/campaigns/Descent_Into_Darkness/scenarios/03_A_Haunting_in_Winter.cfg:1534
 msgid ""
 "Perhaps, but that is not what I meant. Before you can begin to control other "
 "souls, you must first master your own."
 msgstr ""
+"Pravděpodobně ne, ale to jsem nemyslel. Než začneš ovládat jiné duše, musíš "
+"nejprve ovládnout tu svojí."
 
 #. [message]: speaker=Malin Keshar
 #: data/campaigns/Descent_Into_Darkness/scenarios/03_A_Haunting_in_Winter.cfg:1539
 msgid ""
 "What do you mean? I am proud. I am strong. If it is a matter of conviction—"
-msgstr ""
+msgstr "Co tím myslíš? Jsem hrdý. Jsem silný. A pokud jde o přesvědčení..."
 
+# Myslím, že je to přeloženo dobře, ale pořád nějak nechápu tu logiku třetí věty. Nemá tam být, že bariéra mezi životem a smrtí je oslabena? Tedy logicky most je posílen.
```

Comment:
řekl bych, že se jedná o temnou nauku - pokud je smrt násilná, něco ze zemřelého tu zůstává a neodchází a nezmění to ani čas (vizte mou další opravu); pak asi mají mít nekromanti dobrý materiál na znovuoživení
```suggestion
```

Reply:
To už logiku dává. Ale zase to neštimuje v tom druhém použití této věty, kde má Malin nějakou duši vyčiťovat za bariérou/nějakou duši provádět přes bariéru. Jesliže ta duše "neodešla", tam kam by slušné duše měly po smrti odcházet, proč tedy se trápit s nějakou bariérou?
Jinak ta navržená oprava se mi líbí a odsouhlasím jí. Ale tady nebylo asi něco příběhově dobře dotaženo.

Reply:
Slovo barrier je použito jen jednou ve spojení past the barrier. Vnímám to tak, že duše má být hned za bukem, tedy pro nekromanta dosažitelná.

Ve [změně](https://github.com/wesnoth/wesnoth/commit/f72feb9e535cb05bbcc3db02912452ad4e199643), která zavedla barrier a weakened bridge, jsem vysvětlení nenašel.

Můžete se porozhlédnout po hotových překladech do jazyků, kterým vládnete, jestli to někdo pochopil výstižněji.

Já po loňském překládání DiD ve stable to vnímám tak, že princip je i po změně zachován (nekromant cítí duši), jen se prohlubuje řečnění okolo. Cituji (svůj překlad ve) stable:

Duše nedávno zemřelých nebloudí daleko od svých těl. Měl bys být schopen je dosud vycítit.

Vzpomínám si i na dlouhé hledání lepšího překladu názvu scénáře (A Haunting in Winter), protože tam cítím nejméně dva významy, ale nepovedlo se mi to. Tím vás nechci nijak hecovat.

Reply:
Já si to ještě nechám projít hlavou. Dneska už jsme příliš unavený a možná trochu opilý...

Reply:
K tomu názvu scénáře mě to láká udělat nějaký melodramatický oblouk společně s názvem scénáře následujícího. Třeba Zima, která mnohé souží/Jaro pak po pomstě touží. Ale to je na názvy blbé. Možná Zima mnohé souží/Jaro po pomstě touží?

Reply:
Že jsem se neovládl a napsal to...

Mně osobně vadí více jak dvouslovné názvy mapy, protože pak v seznamu uložených her nevidím číslo automatického uložení.

Chtěl jsem najít něco, co by vyjadřovalo, že v jeskyni jsou strašidla (haunted), Malin s Darkenem je loví (haunt) a to vše proto, že se Malin moc chce stát nekromantem (haunting). Ale možná tolik významů za tím názvem není.

Soužení není úplně daleko, ale ta básnička přes dva názvy map mne moc nebere.

Reply:
Otázka temné nauky nejspíš vyřešena. Dá se konvezace o názvu scénáře nějak naklopit do Issue a dodělat schválení stávajících změn?

Reply:
Pořád tam chcete nechat tu poznámku? Mně přijde bariéra vyřešena a poznámka zavádějící (pro dalšího překladatele).

Název scénáře jsem řešit nechtěl, ten začlenění nebrání.